index.js

```diff
@@ -86,7 +86,7 @@ function RingPop(options) {
     this.isReady = false;
 
     this.debugFlags = {};
-    this.joinSize = 3;              // join fanout
+    this.joinSize = options.joinSize;
```

Comment:
consider defaulting this to something or making it a required field. just like app & hostPort are required.